lib/Common/Memory/PageAllocator.cpp

```diff
@@ -2054,9 +2073,8 @@ PageAllocatorBase<TVirtualAlloc, TSegment, TPageSegment>::DecommitNow(bool all)
     PAGE_ALLOC_TRACE_AND_STATS(_u("Free page count = %d"), this->freePageCount);
     PAGE_ALLOC_TRACE_AND_STATS(_u("New free page count = %d"), newFreePageCount);
 
-#if DBG_DUMP
```

Comment:
Should this become #ifdef ENABLE_BASIC_TELEMETRY?

Reply:
i thought it was simpler to just remove it, rather than have another conditional to worry about.  Happy to change it if you feel strongly about it.